webapp-django/crashstats/topcrashers/views.py

```diff
@@ -193,11 +214,14 @@ def topcrashers(request, days=None, possible_days=None, default_context=None):
     context = default_context or {}
 
     product = request.GET.get('product')
-    versions = request.GET.get('version')
+    versions = request.GET.getlist('version')
     crash_type = request.GET.get('process_type')
     os_name = request.GET.get('platform')
     result_count = request.GET.get('_facets_size')
     tcbs_mode = request.GET.get('_tcbs_mode')
+    range_type = request.GET.get('_range_type')
+
+    range_type = range_type == 'build' and 'build' or 'report'
```

Comment:
This looks wrong, but assuming the intention here is to restrict the domain of values to 'build' and 'report' then I think it's correct. To fix the "it looks wrong aspect", I'd probably just add a set of parens like this:

```
range_type = (range_type == 'build') and 'build' or 'report'
```

Though, I think I would have done this with a ternary:

```
range_type = 'build' if range_type == 'build' else 'report'
```